README.md

```diff
@@ -65,6 +65,20 @@ CGFloat myAppBorderRadiusSmall = BPKBorderRadiusSm;
 CGFloat myAppBorderRadiusLarge = BPKBorderRadiusPill;
 ```
 
+### Backpack/Shadow
+
+`Backpack/Shadow` contains the Backpack shadow definitions in the class `BPKShadow`. Add `Backpack/Shadow` to your `Podfile` or `podspec` and then use it as follows.
+
+```objective-c
+#import <Backpack/Shadow.h>
+
+BPKShadow *shadow = [BPKShadow shadowSm];
+// BPKShadow *shadowLg = [BPKShadow shadowLg];
+// BPKShadow *shadowXl = [BPKShadow shadowXl];
```

Comment:
Just want to confirm that these are supposed to be commented out?

Reply:
Yeah